css-brochure-form/styles.css

```diff
@@ -0,0 +1,71 @@
+form {
+  width: 50%;
+  margin: auto;
+}
+
+h3 {
+  text-align: center;
+  color: #ee6c66;
+  text-transform: uppercase;
+  font-family: "Brandon Grotesque", sans-serif;
+  font-style: 900;
+  font-weight: normal;
+  src: url("BrandonGrotesque-Medium.woff2") format("woff2");
```

Comment:
These are only necessary for the `@font-face` rules, you don't need to worry about them. You can just use the font with the `font-family` property.